react-native/android/src/main/java/io/realm/react/util/SSLHelper.java

```diff
@@ -0,0 +1,136 @@
+package io.realm.react.util;
```

Comment:
Please add standard header.

Reply:
@kneth When it's not code changes, I would recommend the reviewer to just fix the issues versus waiting for the submitter to get it done.